src/app/design/design.module.ts

```diff
@@ -82,7 +86,7 @@ import { ModalAddOrderComponent } from './modal-add-order/modal-add-order.compon
         PageHeaderGuidelinesComponent,
         ModalAddOrderComponent,
     ],
-    imports: [CommonModule, FormsModule, DesignRoutingModule, SharedModule, NgbModule, MuziehComponentsModule, MatMenuModule],
-    providers: [{ provide: NAVIGATION_LINK_TOKEN, useValue: navigationLinks }],
+    imports: [CommonModule, FormsModule, DesignRoutingModule, SharedModule, NgbModule, MuziehComponentsModule, MatMenuModule, MatDatepickerModule, MatNativeDateModule, MatFormFieldModule, MatInputModule],
```

Comment:
Menu button, automcomplete, dialog, and datepicker are the only approved material modules to use so far. Nothing else from material should be used.